src/ansys/dpf/post/dpf_solution.py

```diff
@@ -33,25 +36,25 @@ class DpfSolution:
     """Main class of post result API."""
 
     def __init__(self, data_sources, model):
-        """Initialization of the solution using data_sources
-        and dpf.core.Model object."""
+        """Initialize the solution using data_sources and dpf.core.Model object."""
         self._data_sources = data_sources
         self._model = model
 
     @property
     def mesh(self):
-        """Mesh representation of the model.
-        Based on the dpf.core.MeshedRegion class."""
+        """Return the Mesh representation of the model.
+
+        Returns the a :class:`ansys.dpf.core.MeshedRegion`.
+        """
         return self._model.metadata.meshed_region
 
     @property
     def time_freq_support(self):
-        """Description of the temporal/frequency
-        analysis of the model."""
+        """Return the description of the temporal/frequency analysis of the model."""
         return self._model.metadata.time_freq_support
 
     def get_result_info(self):
-        r"""Returns information about the result file.
+        r"""Return information about the result file.
```

Comment:
```suggestion
        """Return information about the result file.
```
Why would the ``r`` be necessary here?